sforzando/q-lako
# Consolidate Initial Processes

## Describe the PR

To close #47 .

## Screenshots

![2020-10-15 00 24 13 localhost 5e3b2dba0cac](https://user-images.githubusercontent.com/32637762/96010492-cd6ae380-0e7c-11eb-9e57-de0b48a945a7.png)

## Detail of the change

- `__init__.py` for consolidation of the initialization process
- `audit_logger.py`
  - Add `delete_test_logs`
- `templates/audit_logs.html`
  - Table background color depends on Log's ID

## Anticipated impacts

None.

## To reproduce

Steps to check the behavior:

1. Normally run `python main.py`
1. Check `http://localhost:8888/` and `http://localhost:8888/audit_logs`
1. Delete `[TEST]` Audit Logs `python audit_logs.py`
1. Reload `http://localhost:8888/audit_logs`

## Additional context

None.
